tests/v2/validation/provisioning/k3s/custom_cluster_test.go

```diff
@@ -76,33 +77,24 @@ func (c *CustomClusterProvisioningTestSuite) SetupSuite() {
 }
 
 func (c *CustomClusterProvisioningTestSuite) TestProvisioningK3SCustomCluster() {
-	nodeRoles0 := []string{
-		"--etcd --controlplane --worker",
-	}
+	nodeRolesAll := []machinepools.NodeRoles{provisioning.AllRolesPool}
```

Comment:
I'm being super picky here...can we have these var be defined with no additional space between them? Looks a bit strange with the extra space